packages/sitecore-jss-react-forms/src/components/field-templates/field-validation-errors.tsx

```diff
@@ -7,7 +7,7 @@ const FieldValidationErrors: React.FunctionComponent<LabelProps> = (props) => {
 
     // strip the errors component from the custom component props
     // (prevents infinite loop rendering if someone reuses this component as a custom error component)
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Hmm I think I need to fix the way in which I'm testing changes locally, because it was giving me errors with the `@typescript-eslint/` and none without. I'll figure that out and revert these.